### DISCUSSION TITLE
client fetch cross-domain

### DISCUSSION THREAD

**Discussion by Lynne42** [remix-run/remix#1035] [Q&A] [answered]
For requests that occur on the browser side, when developing locally, how to proxy the server to handle cross-domain issues

**Comment by sergiodxa** [comment, marked as answer]
The easiest way is to create a [Resource Route](https://remix.run/docs/en/v1/guides/resource-routes) and with the [useFetcher hook](https://remix.run/docs/en/v1/api/remix#usefetcher) load the data from that route.

Then in the loader of your Resource Route you can fetch the URL on the other domain, and a server-to-server request will not have cross-domain issues.
